tst/EnergyPlus/unit/EvaporativeFluidCoolers.unit.cc

```diff
@@ -69,25 +69,7 @@ using namespace EnergyPlus::EvaporativeFluidCoolers;
 
 namespace EnergyPlus {
 
-class EvapFluidCoolersFixture : public EnergyPlusFixture
-{
-public:
-protected:
-    virtual void SetUp()
-    {
-        EnergyPlusFixture::SetUp(); // Sets up the base fixture first.
-
-        state->dataEvapFluidCoolers->NumSimpleEvapFluidCoolers = 1;
-        state->dataEvapFluidCoolers->SimpleEvapFluidCooler.allocate(state->dataEvapFluidCoolers->NumSimpleEvapFluidCoolers);
-    }
-
-    virtual void TearDown()
-    {
-        EnergyPlusFixture::TearDown(); // Remember to tear down the base fixture after cleaning up derived fixture!
-    }
-};
```

Comment:
This turned out to be an unnecessary layer of inheritance for evap fluid cooler testing.